analyzer/tests/functional/analyze_and_parse/test_files/compiler_error.output

```diff
@@ -0,0 +1,37 @@
+NORMAL#CodeChecker log --output $LOGFILE$ --build "make compiler_error" --quiet
+NORMAL#CodeChecker analyze $LOGFILE$ --output $OUTPUT$ --analyzers clang-tidy --quiet
+NORMAL#CodeChecker parse $OUTPUT$
+--------------------------------------------------------------------------------
+[] - Starting static analysis ...
+[] - Analyzing compiler_error.cpp with clang-tidy  failed!
+[] - ----==== Summary ====----
+[] - Failed to analyze
+[] -   clang-tidy: 1
+[] - Total analyzed compilation commands: 1
+[] - ----=================----
+[] - Analysis finished.
+[] - To view results in the terminal use the "CodeChecker parse" command.
+[] - To store results use the "CodeChecker store" command.
+[] - See --help and the user guide for further options about parsing and storing the reports.
+[] - ----=================----
+[MEDIUM] compiler_error.cpp:3:11: expected ';' after return statement [clang-diagnostic-error]
```

Comment:
We set the severity levels of compiler warnings (clang-diagnostic-<warning>) by default to `MEDIUM`. I think it would be better to set these errors to a much higher severity level such as `HIGH`. What do you think?

You can handle diagnostic errors here:
https://github.com/Ericsson/codechecker/blob/9e33f21258f67f6426dcd30cf426e9bdeb6187b1/web/codechecker_web/shared/webserver_context.py#L33-L34

Reply:
Thanks for the feedback. However, I chose "critical" severity instead in case of compiler errors even if its severity could be equivalent with a warning's severity in GCC, indicating that further reports are missing from this file.